### PR TITLE
Fixed compile tests

### PR DESCRIPTION
**Pull request by ahcorde**
Tests are not able to find `resource_retriever/retriever.h` headers

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: transmission_interface/CMakeLists.txt [hunk 1]
```diff
@@ -87,6 +87,7 @@ target_link_libraries(${PROJECT_NAME}_loader_plugins ${PROJECT_NAME}_loader)
 
 if(CATKIN_ENABLE_TESTING)
   find_package(resource_retriever REQUIRED)
+  include_directories(include ${resource_retriever_INCLUDE_DIRS})
```
Comment (matthew-reynolds) [inline transmission_interface/CMakeLists.txt]: Thanks! I definitely missed that in my cleanup of these files... 😬.

We can drop this extra `include`, correct?

```suggestion
  include_directories(${resource_retriever_INCLUDE_DIRS})
```

**Review by matthew-reynolds — CHANGES_REQUESTED** [review]

**Comment by ahcorde** [comment]
thank your for the review @matthew-reynolds

**Comment by bmagyar** [comment]
How did this issue not show up on our CI?

**Comment by matthew-reynolds** [comment]
> How did this issue not show up on our CI?

Probably because `${catkin_INCLUDE_DIRS}` always contains `/opt/ros/melodic/include`.

<sup>(Maybe you could come up with some minimal, contrived, barely-even-ROS example that doesn't include that dir, but in all practical cases, it's always included.)</sup>

**Pull request merged**